### ISSUE TITLE
How to implement the Residual Loss in your code..

### ISSUE THREAD

**Issue by msiraj83** [jackie840129/VDSR-reduction_with-Keras#1]
The VDSR implementation in tensorflow  and caffe have residual loss instead of simple mean square loss. How it  will be implement in keras.

self.loss = tf.reduce_mean(tf.square(self.labels - self.images - self.pred))

The code for the VDSR is available on the following link...

https://github.com/kweisamx/TensorFlow-VDSR/blob/master/model.py

can you help me in this regards ??

**Comment by sivaramakrishnan-rajaraman** [comment]
@msiraj83 did you find a solution for this? Kindly give the function if you did. Thanks.
